lib/perimeterx/utils/px_template_factory.rb

```diff
@@ -11,7 +11,7 @@ def self.get_template(px_ctx, px_config)
       end
 
       logger.debug('PxTemplateFactory[get_template]: rendering template')
-      template_type = px_ctx.context[:block_action] == 'captcha' ? PxModule::CAPTCHA_TEMPLATE : BLOCK_TEMPLATE
+      template_type = px_ctx.context[:block_action] == 'captcha' ? px_config[:captcha_provider] : BLOCK_TEMPLATE
```

Comment:
I sugget to use `.downcase` and rename templates to lowercase, i think this a better convention but we can talk about it

Reply:
Sounds good. Will change.